src/discord/mod.rs

```diff
@@ -14,7 +14,9 @@ use twilight_http::request::channel::reaction::RequestReactionType;
 use twilight_mention::timestamp::{Timestamp as MentionTimestamp, TimestampStyle};
```

Comment:
all these changes are basically just refactoring for fun. which i acknowledge is not great hygiene for a PR but i also dont really care int his context